[2020-11-30 17:34:10,647] SEVERE {b7a.log.crash} - null 
java.lang.NullPointerException
	at org.ballerinalang.net.http.HttpUtil.getErrorMessage(HttpUtil.java:473)
	at org.ballerinalang.net.http.HttpUtil.handleFailure(HttpUtil.java:464)
	at org.ballerinalang.net.http.HttpCallableUnitCallback.notifyFailure(HttpCallableUnitCallback.java:41)
	at io.ballerina.runtime.scheduling.Scheduler.postProcess(Scheduler.java:337)
	at io.ballerina.runtime.scheduling.Scheduler.run(Scheduler.java:293)
	at io.ballerina.runtime.scheduling.Scheduler.runSafely(Scheduler.java:245)
	at java.base/java.lang.Thread.run(Unknown Source)
 
[2020-11-30 17:34:40,453] SEVERE {b7a.log.crash} - null 
java.lang.NullPointerException
	at org.ballerinalang.net.http.HttpUtil.getErrorMessage(HttpUtil.java:473)
	at org.ballerinalang.net.http.HttpUtil.handleFailure(HttpUtil.java:464)
	at org.ballerinalang.net.http.HttpCallableUnitCallback.notifyFailure(HttpCallableUnitCallback.java:41)
	at io.ballerina.runtime.scheduling.Scheduler.postProcess(Scheduler.java:337)
	at io.ballerina.runtime.scheduling.Scheduler.run(Scheduler.java:293)
	at io.ballerina.runtime.scheduling.Scheduler.runSafely(Scheduler.java:245)
	at java.base/java.lang.Thread.run(Unknown Source)
 
[2020-11-30 17:40:14,285] SEVERE {b7a.log.crash} - null 
java.lang.NullPointerException
	at org.ballerinalang.net.http.HttpUtil.getErrorMessage(HttpUtil.java:473)
	at org.ballerinalang.net.http.HttpUtil.handleFailure(HttpUtil.java:464)
	at org.ballerinalang.net.http.HttpCallableUnitCallback.notifyFailure(HttpCallableUnitCallback.java:41)
	at io.ballerina.runtime.scheduling.Scheduler.postProcess(Scheduler.java:337)
	at io.ballerina.runtime.scheduling.Scheduler.run(Scheduler.java:293)
	at io.ballerina.runtime.scheduling.Scheduler.runSafely(Scheduler.java:245)
	at java.base/java.lang.Thread.run(Unknown Source)
 
[2020-11-30 17:46:34,961] SEVERE {b7a.log.crash} - null 
java.lang.NullPointerException
	at org.ballerinalang.net.http.HttpUtil.getErrorMessage(HttpUtil.java:473)
	at org.ballerinalang.net.http.HttpUtil.handleFailure(HttpUtil.java:464)
	at org.ballerinalang.net.http.HttpCallableUnitCallback.notifyFailure(HttpCallableUnitCallback.java:41)
	at io.ballerina.runtime.scheduling.Scheduler.postProcess(Scheduler.java:337)
	at io.ballerina.runtime.scheduling.Scheduler.run(Scheduler.java:293)
	at io.ballerina.runtime.scheduling.Scheduler.runSafely(Scheduler.java:245)
	at java.base/java.lang.Thread.run(Unknown Source)
 
[2020-11-30 17:47:42,202] SEVERE {b7a.log.crash} - null 
java.lang.NullPointerException
	at org.ballerinalang.net.http.HttpUtil.getErrorMessage(HttpUtil.java:473)
	at org.ballerinalang.net.http.HttpUtil.handleFailure(HttpUtil.java:464)
	at org.ballerinalang.net.http.HttpCallableUnitCallback.notifyFailure(HttpCallableUnitCallback.java:41)
	at io.ballerina.runtime.scheduling.Scheduler.postProcess(Scheduler.java:337)
	at io.ballerina.runtime.scheduling.Scheduler.run(Scheduler.java:293)
	at io.ballerina.runtime.scheduling.Scheduler.runSafely(Scheduler.java:245)
	at java.base/java.lang.Thread.run(Unknown Source)
 
[2020-11-30 17:53:06,424] SEVERE {b7a.log.crash} - null 
java.lang.NullPointerException
	at org.ballerinalang.net.http.HttpUtil.getErrorMessage(HttpUtil.java:473)
	at org.ballerinalang.net.http.HttpUtil.handleFailure(HttpUtil.java:464)
	at org.ballerinalang.net.http.HttpCallableUnitCallback.notifyFailure(HttpCallableUnitCallback.java:41)
	at io.ballerina.runtime.scheduling.Scheduler.postProcess(Scheduler.java:337)
	at io.ballerina.runtime.scheduling.Scheduler.run(Scheduler.java:293)
	at io.ballerina.runtime.scheduling.Scheduler.runSafely(Scheduler.java:245)
	at java.base/java.lang.Thread.run(Unknown Source)
 
[2020-11-30 17:53:33,820] SEVERE {b7a.log.crash} - null 
java.lang.NullPointerException
	at org.ballerinalang.net.http.HttpUtil.getErrorMessage(HttpUtil.java:473)
	at org.ballerinalang.net.http.HttpUtil.handleFailure(HttpUtil.java:464)
	at org.ballerinalang.net.http.HttpCallableUnitCallback.notifyFailure(HttpCallableUnitCallback.java:41)
	at io.ballerina.runtime.scheduling.Scheduler.postProcess(Scheduler.java:337)
	at io.ballerina.runtime.scheduling.Scheduler.run(Scheduler.java:293)
	at io.ballerina.runtime.scheduling.Scheduler.runSafely(Scheduler.java:245)
	at java.base/java.lang.Thread.run(Unknown Source)
 
[2020-12-01 11:59:15,427] SEVERE {b7a.log.crash} - error("Error in SQL connector configuration: Failed to get driver instance for jdbcUrl=jdbc:mysql://localhost:3306/CustomerDB?serverTimezone=UTC Caused by :No suitable driver") 
io.ballerina.runtime.util.exceptions.BallerinaException: error("Error in SQL connector configuration: Failed to get driver instance for jdbcUrl=jdbc:mysql://localhost:3306/CustomerDB?serverTimezone=UTC Caused by :No suitable driver")
	at org.ballerinalang.test.runtime.entity.TesterinaFunction.runOnSchedule(TesterinaFunction.java:122)
	at org.ballerinalang.test.runtime.entity.TesterinaFunction.invoke(TesterinaFunction.java:56)
	at org.ballerinalang.test.runtime.BTestRunner.startSuite(BTestRunner.java:562)
	at org.ballerinalang.test.runtime.BTestRunner.execute(BTestRunner.java:288)
	at org.ballerinalang.test.runtime.BTestRunner.runTest(BTestRunner.java:107)
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:52)
Caused by: error("Error in SQL connector configuration: Failed to get driver instance for jdbcUrl=jdbc:mysql://localhost:3306/CustomerDB?serverTimezone=UTC Caused by :No suitable driver")
	at io.ballerina.runtime.api.ErrorCreator.createDistinctError(ErrorCreator.java:154)
	at io.ballerina.runtime.api.ErrorCreator.createDistinctError(ErrorCreator.java:138)
	at org.ballerinalang.sql.utils.ErrorGenerator.getSQLApplicationError(ErrorGenerator.java:65)
	at org.ballerinalang.sql.datasource.SQLDatasource.buildDataSource(SQLDatasource.java:242)
	at org.ballerinalang.sql.datasource.SQLDatasource.<init>(SQLDatasource.java:55)
	at org.ballerinalang.sql.datasource.SQLDatasource.createAndInitDatasource(SQLDatasource.java:119)
	at org.ballerinalang.sql.datasource.SQLDatasource.lambda$retrieveDatasource$1(SQLDatasource.java:112)
	at java.base/java.util.concurrent.ConcurrentHashMap.computeIfAbsent(Unknown Source)
	at org.ballerinalang.sql.datasource.SQLDatasource.retrieveDatasource(SQLDatasource.java:111)
	at org.ballerinalang.sql.utils.ClientUtils.createClient(ClientUtils.java:43)
	at org.ballerinalang.jdbc.NativeImpl.createClient(NativeImpl.java:75)
	at ballerina.java_jdbc.0_5_1.client.createClient(client.bal:147)
	at ballerina.java_jdbc.0_5_1.$value$Client.init(client.bal:44)
	at ballerina.java_jdbc.0_5_1.$value$Client.call(client.bal)
	at ballerina.java_jdbc.0_5_1.$value$Client.$init$(client.bal:22)
	at ballerina.java_jdbc.0_5_1.$value$Client.call(client.bal)
	at aneesha.RESTModuleDB.0_1_0.RESTModuleDB.$aneesha_RESTModuleDB_0_1_0__init_0(RESTModuleDB:12)
	at aneesha.RESTModuleDB.0_1_0.$_init.$aneesha_RESTModuleDB_0_1_0__init_(RESTModuleDB:10)
	at aneesha.RESTModuleDB.0_1_0.$_init.$moduleInit(RESTModuleDB)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at org.ballerinalang.test.runtime.entity.TesterinaFunction.lambda$runOnSchedule$0(TesterinaFunction.java:109)
	at io.ballerina.runtime.scheduling.SchedulerItem.execute(Scheduler.java:501)
	at io.ballerina.runtime.scheduling.Scheduler.run(Scheduler.java:277)
	at io.ballerina.runtime.scheduling.Scheduler.runSafely(Scheduler.java:245)
	at java.base/java.lang.Thread.run(Unknown Source)
 
[2020-12-01 12:00:52,112] SEVERE {b7a.log.crash} - error("Error in SQL connector configuration: Failed to get driver instance for jdbcUrl=jdbc:mysql://localhost:3306/CustomerDB?serverTimezone=UTC Caused by :No suitable driver") 
io.ballerina.runtime.util.exceptions.BallerinaException: error("Error in SQL connector configuration: Failed to get driver instance for jdbcUrl=jdbc:mysql://localhost:3306/CustomerDB?serverTimezone=UTC Caused by :No suitable driver")
	at org.ballerinalang.test.runtime.entity.TesterinaFunction.runOnSchedule(TesterinaFunction.java:122)
	at org.ballerinalang.test.runtime.entity.TesterinaFunction.invoke(TesterinaFunction.java:56)
	at org.ballerinalang.test.runtime.BTestRunner.startSuite(BTestRunner.java:562)
	at org.ballerinalang.test.runtime.BTestRunner.execute(BTestRunner.java:288)
	at org.ballerinalang.test.runtime.BTestRunner.runTest(BTestRunner.java:107)
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:52)
Caused by: error("Error in SQL connector configuration: Failed to get driver instance for jdbcUrl=jdbc:mysql://localhost:3306/CustomerDB?serverTimezone=UTC Caused by :No suitable driver")
	at io.ballerina.runtime.api.ErrorCreator.createDistinctError(ErrorCreator.java:154)
	at io.ballerina.runtime.api.ErrorCreator.createDistinctError(ErrorCreator.java:138)
	at org.ballerinalang.sql.utils.ErrorGenerator.getSQLApplicationError(ErrorGenerator.java:65)
	at org.ballerinalang.sql.datasource.SQLDatasource.buildDataSource(SQLDatasource.java:242)
	at org.ballerinalang.sql.datasource.SQLDatasource.<init>(SQLDatasource.java:55)
	at org.ballerinalang.sql.datasource.SQLDatasource.createAndInitDatasource(SQLDatasource.java:119)
	at org.ballerinalang.sql.datasource.SQLDatasource.lambda$retrieveDatasource$1(SQLDatasource.java:112)
	at java.base/java.util.concurrent.ConcurrentHashMap.computeIfAbsent(Unknown Source)
	at org.ballerinalang.sql.datasource.SQLDatasource.retrieveDatasource(SQLDatasource.java:111)
	at org.ballerinalang.sql.utils.ClientUtils.createClient(ClientUtils.java:43)
	at org.ballerinalang.jdbc.NativeImpl.createClient(NativeImpl.java:75)
	at ballerina.java_jdbc.0_5_1.client.createClient(client.bal:147)
	at ballerina.java_jdbc.0_5_1.$value$Client.init(client.bal:44)
	at ballerina.java_jdbc.0_5_1.$value$Client.call(client.bal)
	at ballerina.java_jdbc.0_5_1.$value$Client.$init$(client.bal:22)
	at ballerina.java_jdbc.0_5_1.$value$Client.call(client.bal)
	at aneesha.RESTModuleDB.0_1_0.RESTModuleDB.$aneesha_RESTModuleDB_0_1_0__init_0(RESTModuleDB:12)
	at aneesha.RESTModuleDB.0_1_0.$_init.$aneesha_RESTModuleDB_0_1_0__init_(RESTModuleDB:10)
	at aneesha.RESTModuleDB.0_1_0.$_init.$moduleInit(RESTModuleDB)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at org.ballerinalang.test.runtime.entity.TesterinaFunction.lambda$runOnSchedule$0(TesterinaFunction.java:109)
	at io.ballerina.runtime.scheduling.SchedulerItem.execute(Scheduler.java:501)
	at io.ballerina.runtime.scheduling.Scheduler.run(Scheduler.java:277)
	at io.ballerina.runtime.scheduling.Scheduler.runSafely(Scheduler.java:245)
	at java.base/java.lang.Thread.run(Unknown Source)
 
[2020-12-01 12:10:38,700] SEVERE {b7a.log.crash} - error("Error in SQL connector configuration: Failed to get driver instance for jdbcUrl=jdbc:mysql://localhost:3306/CustomerDB?serverTimezone=UTC Caused by :No suitable driver") 
io.ballerina.runtime.util.exceptions.BallerinaException: error("Error in SQL connector configuration: Failed to get driver instance for jdbcUrl=jdbc:mysql://localhost:3306/CustomerDB?serverTimezone=UTC Caused by :No suitable driver")
	at org.ballerinalang.test.runtime.entity.TesterinaFunction.runOnSchedule(TesterinaFunction.java:122)
	at org.ballerinalang.test.runtime.entity.TesterinaFunction.invoke(TesterinaFunction.java:56)
	at org.ballerinalang.test.runtime.BTestRunner.startSuite(BTestRunner.java:562)
	at org.ballerinalang.test.runtime.BTestRunner.execute(BTestRunner.java:288)
	at org.ballerinalang.test.runtime.BTestRunner.runTest(BTestRunner.java:107)
	at org.ballerinalang.test.runtime.util.TesterinaUtils.executeTests(TesterinaUtils.java:74)
	at org.ballerinalang.test.runtime.Main.startTestSuit(Main.java:60)
	at org.ballerinalang.test.runtime.Main.main(Main.java:52)
Caused by: error("Error in SQL connector configuration: Failed to get driver instance for jdbcUrl=jdbc:mysql://localhost:3306/CustomerDB?serverTimezone=UTC Caused by :No suitable driver")
	at io.ballerina.runtime.api.ErrorCreator.createDistinctError(ErrorCreator.java:154)
	at io.ballerina.runtime.api.ErrorCreator.createDistinctError(ErrorCreator.java:138)
	at org.ballerinalang.sql.utils.ErrorGenerator.getSQLApplicationError(ErrorGenerator.java:65)
	at org.ballerinalang.sql.datasource.SQLDatasource.buildDataSource(SQLDatasource.java:242)
	at org.ballerinalang.sql.datasource.SQLDatasource.<init>(SQLDatasource.java:55)
	at org.ballerinalang.sql.datasource.SQLDatasource.createAndInitDatasource(SQLDatasource.java:119)
	at org.ballerinalang.sql.datasource.SQLDatasource.lambda$retrieveDatasource$1(SQLDatasource.java:112)
	at java.base/java.util.concurrent.ConcurrentHashMap.computeIfAbsent(Unknown Source)
	at org.ballerinalang.sql.datasource.SQLDatasource.retrieveDatasource(SQLDatasource.java:111)
	at org.ballerinalang.sql.utils.ClientUtils.createClient(ClientUtils.java:43)
	at org.ballerinalang.jdbc.NativeImpl.createClient(NativeImpl.java:75)
	at ballerina.java_jdbc.0_5_1.client.createClient(client.bal:147)
	at ballerina.java_jdbc.0_5_1.$value$Client.init(client.bal:44)
	at ballerina.java_jdbc.0_5_1.$value$Client.call(client.bal)
	at ballerina.java_jdbc.0_5_1.$value$Client.$init$(client.bal:22)
	at ballerina.java_jdbc.0_5_1.$value$Client.call(client.bal)
	at aneesha.RESTModuleDB.0_1_0.RESTModuleDB.$aneesha_RESTModuleDB_0_1_0__init_0(RESTModuleDB:12)
	at aneesha.RESTModuleDB.0_1_0.$_init.$aneesha_RESTModuleDB_0_1_0__init_(RESTModuleDB:10)
	at aneesha.RESTModuleDB.0_1_0.$_init.$moduleInit(RESTModuleDB)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at org.ballerinalang.test.runtime.entity.TesterinaFunction.lambda$runOnSchedule$0(TesterinaFunction.java:109)
	at io.ballerina.runtime.scheduling.SchedulerItem.execute(Scheduler.java:501)
	at io.ballerina.runtime.scheduling.Scheduler.run(Scheduler.java:277)
	at io.ballerina.runtime.scheduling.Scheduler.runSafely(Scheduler.java:245)
	at java.base/java.lang.Thread.run(Unknown Source)
 
